[BUG] Semicolon error while compiling

Hi, I tried to install it on my WSL 1 machine. However, it failed in compile process due to a semicolon error. It think it is related to macros.

`In file included from /home/murat/benchmark/benchmark/googletest/googletest/src/gtest-all.cc:38:0:                                                                     /home/murat/benchmark/benchmark/googletest/googletest/include/gtest/gtest.h:79:45: error: extra ‘;’ [-Wpedantic]                                                        GTEST_DECLARE_bool_(also_run_disabled_tests);`

**System**
Which OS, compiler, and compiler version are you using:
  - OS: Ubuntu 18.04 WSL1
  - Compiler and version: `gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0` - `cmake version 3.10.2`
  
I have only followed the steps given in readme.md.

Thanks in advance.

--- a/cmake/GoogleTest.cmake
+++ b/cmake/GoogleTest.cmake
@@ -29,6 +29,9 @@ set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
 include(${GOOGLETEST_PREFIX}/googletest-paths.cmake)
 
+# TODO(dominic): See #1202. Remove if https://github.com/google/googletest/pull/3492 lands upstream.
+add_compile_options(-w)
+
 # Add googletest directly to our build. This defines
 # the gtest and gtest_main targets.
 add_subdirectory(${GOOGLETEST_SOURCE_DIR}

